src/auth0-session/handlers/callback.ts

```diff
@@ -15,6 +15,8 @@ export type AfterCallback = (req: any, res: any, session: any, state: Record<str
 
 export type CallbackOptions = {
   afterCallback?: AfterCallback;
+
+  redirectUri?: string;
```

Comment:
Can you also add this option to https://github.com/auth0/nextjs-auth0/blob/main/src/handlers/callback.ts#L71-L73 ?